dev/release/verify-release-candidate.sh

```diff
@@ -246,38 +246,42 @@ install_dotnet() {
     return 0
   fi
 
-  show_info "Ensuring that .NET is installed..."
-
-  if dotnet --version | grep 8\.0 > /dev/null 2>&1; then
-    local csharp_bin=$(dirname $(which dotnet))
-    show_info "Found C# at $(which csharp) (.NET $(dotnet --version))"
-  else
-    if which dotnet > /dev/null 2>&1; then
-      show_info "dotnet found but it is the wrong version and will be ignored."
+  if command -v dotnet; then
+    show_info "Found $(dotnet --version) at $(which dotnet)"
+
+    if dotnet --version | grep 8\.0 > /dev/null 2>&1; then
```

Comment:
FYI: We can use `--fixed-string`/`-F` for this case and we can use `--quiet` instead of redirect:

```suggestion
    if dotnet --version | grep --quiet --fixed-string 8.0; then
```